modules/op-performance-tuning-using-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,73 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-performance-tuning-using-tektonconfig-cr_{context}"]
+= Performance tuning using TektonConfig CR
+
+You can modify the fields under the `.spec.pipeline.performance` parameter in the `TektonConfig` custom resource (CR) to change high availability (HA) support and performance configuration for the {pipelines-shortname} controller.
+
+.Example: `TektonConfig` performance fields
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    performance:
+      disable-ha: false
+      buckets: 1
+      threads-per-controller: 2
+      kube-api-qps: 5.0
+      kube-api-burst: 10
+----
+
+The fields are optional. If you set them, the {pipelines-title} Operator includes most of the fields as arguments in the `openshift-pipelines-controller` deployment under the `openshift-pipelines-controller` container. The {pipelines-shortname} Operator also updates the `buckets` field in the `config-leader-election` configuration map under the `openshift-pipelines` namespace.
+
+If you do not specify the values, the {pipelines-shortname} Operator does not update those fields and applies the default values for the {pipelines-shortname} controller.
+
+[NOTE]
+====
+If you modify or remove any of the performance fields, the {pipelines-shortname} Operator updates the `openshift-pipelines-controller` deployment and the `config-leader-election` configuration map (if the `buckets` field changed), and recreates `openshift-pipelines-controller` pods.
+====
+
+The following are the modifiable fields:
+
+[options="header"]
+|===
+
+| Name | Description | Default value for the pipeline controller
+
+| `disable-ha` | Enable or disable the HA support. By default, the HA support is enabled. | `false`
+
+| `buckets` | Indicates the number of buckets used to partition key space of each reconciler.
+
+The given number of replicas uses the specified number of buckets. The owner of a bucket handles the reconciling of the keys partitioned into that bucket. The maximum value is `10` | `1`
```

Comment:
Could we still expand "owner" to "the instance that owns"? Usually "owner" is a user